src/app/space/forge-wizard/components/pipeline-view/pipeline-view.component.less

```diff
@@ -0,0 +1,207 @@
+@import (reference) '../../../../../assets/stylesheets/shared/osio.less';
+@icon-path: '../../../../../assets/stylesheets/shared/_icons.less';
+:host {
+  .pipeline {
+    padding: .25rem;
+    border: 0 solid;
+    border-color: @color-pf-black-400;
+    background-color: @color-pf-white;
+    display: inline-flex;
```

Comment:
Adding flex-wrap: wrap here makes it so that a long pipeline doesn't extend past the edge of the wizard. (I am not sure if there are other side effects of this)